packages/docs/docs/concepts/cornerstone-tools/touch.md

```diff
@@ -0,0 +1,218 @@
+  /** The current touch position. */
+  currentPointsList: ITouchPoints[],
+
+  /** The difference between the current and last points. */
+  deltaPoints: IPoints,
+  /** The difference between distances between the current and last points. */
+  deltaDistance: IDistance,
+};
+```
+
+`deltaPoints` is the difference between the mean coordinate point of `lastPointsList` and `currentPointsList`.
+`deltaDistance` is the difference between the average distance between points in `lastPointsList` vs `currentPointsList`
+
+## Usage
+
+You can add an event listener to the element for the event.
+
+```js
+import Events from '@cornerstonejs/tools/enums/Events';
+// element is the cornerstone viewport element
+element.addEventListener(Events.TOUCH_DRAG, (evt) => {
+  // my function on drag
+  console.log(evt);
+});
+
+element.addEventListener(Events.TOUCH_SWIPE, (evt) => {
+  // my function on swipe
+  console.log(evt);
+});
+
+// within the chrome console in a deployed OHIF application
+cornerstone
+  .getEnabledElements()[0]
+  .viewport.element.addEventListener(Events.TOUCH_SWIPE, (evt) => {
+    // my function on swipe
+    console.log('SWIPE', evt);
+  });
+```
+
+A full example can be found by running
+`yarn run example stackManipulationToolsTouch` whose source is [here](https://github.com/gradienthealth/cornerstone3D-beta/blob/gradienthealth/added_touch_events/packages/tools/examples/stackManipulationToolsTouch/index.ts)
+
+## Binding
+
+Touch tools have bindings depending on the number of pointers that are placed down.
+In the future, bindings can be filter based on force, as well as radius (stylus detection).
+The `numTouchPoints` can be as many as is supported by hardware.
+
+```js
+// Add tools to Cornerstone3D
+cornerstoneTools.addTool(PanTool);
+cornerstoneTools.addTool(WindowLevelTool);
+cornerstoneTools.addTool(StackScrollTool);
+cornerstoneTools.addTool(ZoomTool);
+
+// Define a tool group, which defines how mouse events map to tool commands for
+// Any viewport using the group
+const toolGroup = ToolGroupManager.createToolGroup(toolGroupId);
+
+// Add tools to the tool group
+toolGroup.addTool(WindowLevelTool.toolName);
+toolGroup.addTool(PanTool.toolName);
+toolGroup.addTool(ZoomTool.toolName);
+toolGroup.addTool(StackScrollTool.toolName);
+
+// Set the initial state of the tools, here all tools are active and bound to
+// Different touch inputs
+// 5 touch points are possible => unlimited touch points are supported, but is generally limited by hardware.
+toolGroup.setToolActive(ZoomTool.toolName, {
+  bindings: [{ numTouchPoints: 2 }],
+});
+toolGroup.setToolActive(StackScrollTool.toolName, {
+  bindings: [{ numTouchPoints: 3 }],
+});
+toolGroup.setToolActive(WindowLevelTool.toolName, {
+  bindings: [
+    {
+      mouseButton: MouseBindings.Primary, // special condition for one finger touch
+    },
+  ],
+});
+```
+
+The `MouseBindings.Primary` is a special binding type which will
+automatically bind single finger touch.
```

Comment:
I don't understand, why the API here is different, is this supposed to do a mouse binding or touch bindings? If it is touch, I expected the following for consistensy

```
toolGroup.setToolActive(WindowLevelTool.toolName, {
  bindings: [{ numTouchPoints: 1 }],
});
```